doc/robot_model_and_robot_state/src/robot_model_and_robot_state_tutorial.cpp

```diff
@@ -34,18 +34,31 @@
 
 /* Author: Sachin Chitta, Michael Lautman*/
 
-#include <ros/ros.h>
-
 // MoveIt
 #include <moveit/robot_model_loader/robot_model_loader.h>
 #include <moveit/robot_model/robot_model.h>
 #include <moveit/robot_state/robot_state.h>
 
+// All source files that use ROS logging should define a file-specific
+// static const rclcpp::Logger named LOGGER, located at the top of the file
+// and inside the namespace with the narrowest scope (if there is one)
```

Comment:
I think this recommendation is too specific, we might not ALWAYS do it that way. For example, you could do `node->get_logger()` instead. That seems just as valid.

```suggestion
// Often a static const rclcpp::Logger named LOGGER is defined at the top of the file
// and inside the namespace with the narrowest scope (if there is one)
```

Reply:
I agree, I was just copying his from another tutorial. I am 100% in favor of  **node->get_logger()**